src/main/java/pages/TestPage.java

```diff
@@ -33,25 +34,15 @@ public void addTestType() {
     }
 
     public boolean isPageTitleDisplayed() {
-        boolean status = false;
-        if (findElementById(SELECT_PREPARER_PAGE_TITLE).isDisplayed())
-            status = true;
-        return status;
+        return findElementById(SELECT_PREPARER_PAGE_TITLE).isDisplayed();
     }
 
     public boolean isVehicleRegistrationPlateDisplayed(String regPlate) {
-        boolean status = false;
-        if (findElementByXpath("//XCUIElementTypeButton[@name='" + regPlate + " (PSV) 1B7GG36N12S678410 Details arrow forward']").isDisplayed()) {
-            status = true;
-        }
-        return status;
+        return findElementByXpath("//XCUIElementTypeButton[@name='" + regPlate + " (PSV) 1B7GG36N12S678410 Details arrow forward']").isDisplayed();
```

Comment:
please add 1B7GG36N12S678410 as parameter ( String vin ) and pass it through test